react/fields/MonthPicker/CustomMonthPicker/CustomMonthPicker.js

```diff
@@ -0,0 +1,168 @@
+
+const getYearOptions = () => {
+  const maxYear = new Date().getFullYear();
+  const minYear = maxYear - 100;
+
+  return range(maxYear, minYear - 1).map(value => {
+    const stringValue = `${value}`;
+
+    return {
+      value: stringValue,
+      label: stringValue
+    };
+  });
+};
+
+const months = [
+  { value: '1', label: 'Jan' },
+  { value: '2', label: 'Feb' },
+  { value: '3', label: 'Mar' },
+  { value: '4', label: 'Apr' },
+  { value: '5', label: 'May' },
+  { value: '6', label: 'Jun' },
+  { value: '7', label: 'Jul' },
+  { value: '8', label: 'Aug' },
+  { value: '9', label: 'Sep' },
+  { value: '10', label: 'Oct' },
+  { value: '11', label: 'Nov' },
+  { value: '12', label: 'Dec' }
+];
+const years = getYearOptions();
+
+export default class CustomMonthPicker extends Component {
+
+  static displayName = 'CustomMonthPicker';
+
+  static propTypes = {
+    onChange: PropTypes.func,
+    onBlur: PropTypes.func,
+    value: PropTypes.shape({
+      month: PropTypes.number,
+      year: PropTypes.number
+    }),
+    invalid: PropTypes.bool,
+    className: PropTypes.string,
+    id: PropTypes.string
+  };
+
+  static defaultProps = {
+    invalid: false,
+    value: {}
+  };
+
+  constructor() {
+    super();
+
+    this.handleMonthChange = this.handleMonthChange.bind(this);
+    this.handleYearChange = this.handleYearChange.bind(this);
+    this.storeMonthReference = this.storeMonthReference.bind(this);
+    this.storeYearReference = this.storeYearReference.bind(this);
+    this.handleBlur = this.handleBlur.bind(this);
+    this.blurIfNotFocussed = this.blurIfNotFocussed.bind(this);
+  }
+
+  storeMonthReference(input) {
+    if (input !== null) {
+      this.monthInput = input;
+    }
+  }
+
+  storeYearReference(input) {
+    if (input !== null) {
+      this.yearInput = input;
+    }
+  }
+
+  handleMonthChange({ target: { value } }) {
+    const { onChange, value: { year } } = this.props;
+
+    onChange({
+      month: parseInt(value, 10),
+      year
+    });
+  }
+
+  handleYearChange({ target: { value } }) {
+    const { onChange, value: { month } } = this.props;
+
+    onChange({
+      month,
+      year: parseInt(value, 10)
+    });
+  }
+
+  blurIfNotFocussed(active) {
+    const { onBlur, value } = this.props;
+
+    if (active !== this.monthInput && active !== this.yearInput) {
+      onBlur(value);
+    }
+  }
+
+  handleBlur({ relatedTarget }) {
+    const { onBlur } = this.props;
+
+    if (onBlur) {
+      if (relatedTarget !== null) {
+        this.blurIfNotFocussed(relatedTarget);
+      } else {
+        // IE 9 - 11 Hack -- relatedTarget is null in blur event
+        setTimeout(() => {
+          const { activeElement } = document;
+          this.blurIfNotFocussed(activeElement);
+        });
+      }
+    }
+  }
+
+  render() {
+    const { value, className, invalid, id } = this.props;
+    const { month, year } = value;
+    const monthValue = month ? `${month}` : '';
```

Comment:
for the same reason [as above](https://github.com/SEEK-Jobs/seek-style-guide/pull/64/files#r87528592), `String(month || '')`? Depending on your thoughts on the previous comment